main.go

```diff
@@ -14,191 +16,328 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const concurrentFetch = 100
+
+// Commandline flags.
 var (
-	configFile  = flag.String("config.file", "", "Path to config file.")
-	metricsPath = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	addr           = flag.String("web.listen-address", ":9105", "Address to listen on for web interface and telemetry")
+	autoDiscover   = flag.Bool("exporter.discovery", false, "Discover all Mesos slaves")
+	localAddr      = flag.String("exporter.local-address", "127.0.0.1:5051", "Address to connect to the local Mesos slave")
```

Comment:
Ok, last thing :) I noticed that `exporter.local-address` is specified as a host:port address, while `exporter.discovery.master` is a full URL. Would it make sense to also specify the local slave address as a URL? In the end, it's doing HTTP, right?